<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>relocationDialog</class>
 <widget class="QDialog" name="relocationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="habitantLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>живёт в </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="curRoomLabel">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="toRoomL">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Переместить в</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="roomNumBox">
           <property name="minimumSize">
            <size>
             <width>79</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>79</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>relocationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>relocationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
